girder/api/v1/file.py

```diff
@@ -266,7 +269,14 @@ def download(self, file, params):
             if endByte is not None:
                 endByte = int(endByte)
 
-        return self.model('file').download(file, offset, endByte=endByte)
+        contentDisp = params.get('contentDisposition', None)
+        allowed = ['inline', 'attachment']
```

Comment:
``allowed`` should be a set (just define the literal inside curly braces). You can just inline the variable too.